Sources/SQLite/Core/Connection.swift

```diff
@@ -152,6 +152,23 @@ public final class Connection {
         Int(sqlite3_total_changes(handle))
     }
 
+    /// Gets the user version of the database.
+    /// See SQLite [PRAGMA user_version](https://sqlite.org/pragma.html#pragma_user_version)
+    /// - Returns: the user version of the database
+    public func getUserVersion() throws -> Int32? {
```

Comment:
```
var userVersion: Int32? {
   get { }
   set { }
}
```

Reply:
I tried that first, but I ended with problems with `try`. I can eventually ignore them with optionals

Reply:
Yes, I think it's ok to ignore the error here